cmd/mdatagen/internal/metadata.go

```diff
@@ -338,3 +343,52 @@ func (a Attribute) TestValue() string {
 	}
 	return ""
 }
+
+func (a Attribute) TestValueTemplateKey() string {
```

Comment:
Could this function use the output of `TestResultTemplateValue` below to fill in the values?